app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -10,8 +10,8 @@
 <%= render "govuk_publishing_components/components/contextual_guidance", {
   id: field.id,
   html_for: "#{field.id}-field",
-  title: edition.document_type.guidance_for(field.id).title,
-  content: document_contents_guidance
+  title: edition.document_type.guidance_for(field.id) ? edition.document_type.guidance_for(field.id).title : nil,
+  content: edition.document_type.guidance_for(field.id) ? document_contents_guidance : nil
```

Comment:
Instead of inline ternaries for every field, I'd be tempted to create the hash in one go:

```erb
<%
  field_contextual_guidance = nil
  if edition.document_type.guidance_for(field.id)
    field_contextual_guidance = {
      id: field.id,
      html_for: "...etc"
    }
  end
```

Then it's just:

```erb
<%= render "govuk_publishing_components/components/contextual_guidance", field_contextual_guidance do %>
  // render markdown editor
<% end %>
```

If `id` and `html_for` are required - even when no contextual guidance is rendered - I'd suggest removing that requirement from the component (ideally), or else just switching to a merge strategy:

```erb
<%
  field_contextual_guidance = { id: field.id, html_for: "...etc" }
  if edition.document_type.guidance_for(field.id)
    field_contextual_guidance.merge!({
      title: etc...
    })
  end
```

Reply:
I usually run `git rebase --ignore-date master` to reset the time on each commit and have them appear in the right order.